docs/src/main/sphinx/installation/plugins.md

```diff
@@ -0,0 +1,311 @@
+# Plugins 
+
+Trino uses a plugin architecture to extend its capabilities and integrate with
+various data sources and other systems. Trino includes many plugins as part of
+the binary packages - specifically the `tar.gz` archive and the Docker
+container.
+
+Plugins implement some of the following capabilities:
+
+* [Connectors](/connector),
+* [Authentication types](security-authentication)
+* [Access control systems](security-access-control)
+* [Event listeners](admin-event-listeners)
+* Additional types, functions, and table functions
+* Block encodings
+* Resource group configuration managers
+* Session property configuration managers
+* Exchange managers
+* Spooling managers
+
+All plugins are optional for your use of Trino because they support a specific
+functionality, that is potentially not needed for your use case. Plugins are
+located in the `plugin` folder of your Trino installation and are loaded
+automatically during Trino startup.
+
+(plugins-download)=
+## Download
+
+Typically, downloading a plugin is not necessary because Trino binaries include
+many plugins as part of the binary packages.
+
+Every Trino release publishes each plugin as a ZIP archive to the [Maven Central
+Repository](https://central.sonatype.com/). Refer to [plugins-list] for details.
+The specific location is derived from the Maven coordinates of each plugin as
+defined in the `pom.xml` of the source code for the plugin.
+
+For example, the PostgreSQL connector plugin can be found in the
+`plugin/trino-postgresql` directory, and the `pom.xml` file contains the
+following identifier section:
+
+```xml
+<parent>
+    <groupId>io.trino</groupId>
+    <artifactId>trino-root</artifactId>
+    <version>470</version>
+    <relativePath>../../pom.xml</relativePath>
+</parent>
+
+<artifactId>trino-postgresql</artifactId>
+<packaging>trino-plugin</packaging>
+```
+
+The Maven coordinates are therefore `io.trino:trino-postgresql:470` with version
+or `io.trino:trino-postgresql` without version. Use this term for a [search to
+locate the
+artifact](https://central.sonatype.com/search?q=io.trino%3Atrino-postgresql).
```

Comment:
It's actually hard to find the ZIP file from the search landing page. Should we add instructions?

> After searching, click *"View all"* next to *"Latest version"*, then click *"Browse"* to find the ZIP file for the desired version.

Reply:
I will probably add a link to each page .. at least to each one we make optional. But that will be in follow up PRs. Do you think we want a link for all plugins or just optional ones.

Reply:
Added